pkg/interfaces/trigger.go

```diff
@@ -40,9 +39,6 @@ type Trigger interface {
 }
 
 // TriggerMessageProcessor provides an interface that can be used by custom triggers to invoke the runtime
-type TriggerMessageProcessor func(ctx AppFunctionContext, envelope types.MessageEnvelope) error
-
-// TriggerContextBuilder provides an interface to construct an AppFunctionContext for message
-type TriggerContextBuilder func(env types.MessageEnvelope) AppFunctionContext
+type TriggerMessageProcessor func(envelope types.MessageEnvelope) error
```

Comment:
This is a breaking change to the Trigger API. 
Add new version, say `RunMessageProcessor `,  and mark others as deprecated.